manager-bundle/src/Resources/skeleton/config/config.yml

```diff
@@ -57,29 +63,23 @@ doctrine:
         naming_strategy: doctrine.orm.naming_strategy.default
         metadata_cache_driver:
             type: service
-            id: doctrine_cache.providers.doctrine_metadata_cache
-        result_cache_driver:
-            type: service
-            id: doctrine_cache.providers.doctrine_result_cache
+            id: doctrine.system_cache_provider
         query_cache_driver:
             type: service
-            id: doctrine_cache.providers.doctrine_query_cache
+            id: doctrine.system_cache_provider
+        result_cache_driver:
+            type: service
+            id: doctrine.result_cache_provider
 
-# Doctrine cache configuration
-doctrine_cache:
-    providers:
-        doctrine_metadata_cache:
-            file_system:
-                extension: .cache
-                directory: '%kernel.cache_dir%/doctrine/metadata'
-        doctrine_result_cache:
-            file_system:
-                extension: .cache
-                directory: '%kernel.cache_dir%/doctrine/result'
-        doctrine_query_cache:
-            file_system:
-                extension: .cache
-                directory: '%kernel.cache_dir%/doctrine/query'
+services:
```

Comment:
@contao/developers Not sure if you are OK with having this here? We could also move it to a `services.yml` file.

Reply:
It certainly must be in the `services.yml`. And the service name should be `contao_manager.doctrine_result_cache` or similar, because the vendor prefix should be of the bundle.

Reply:
> It certainly must be in the `services.yml`.

Agreed.

> And the service name should be `contao_manager.doctrine_result_cache` or similar, because the vendor prefix should be of the bundle.

Not sure about that. The code is taken from the official Symfony recipe and assumes that the cache is added on application level. The `doctrine` prefix seems correct to me.

Reply:
Changed as discussed in 0b67fbd2e31172b27a48157a24a79cc8cdec1284.